doc/devel/pr_guide.rst

```diff
@@ -176,11 +176,22 @@ a corresponding branch.
 
 Merging
 -------
+As a guiding principle, we require two approvals from core developers before
+merging a pull request. This two-pairs-of-eyes strategy shall ensure a
+consistent project direction and prevent accidental mistakes. It's permissible
+to merge with one approval if the change is not fundamental and can easily be
+reverted any time in the future.
 
-* Documentation and examples may be merged by the first reviewer.  Use
+Some explicit rules following from this:
+
+* *Documentation and examples* may be merged by the first reviewer.  Use
   the threshold "is this better than it was?" as the review criteria.
 
-* For code changes (anything in ``src`` or ``lib``) at least two
+* Minor *infrastructure updates*, e.g. temporary pinning of broken dependencies
+  or small changes to the CI configuration, can be merged with a single
+  approval.
+
+* *Code changes* (anything in ``src`` or ``lib``) need at least two
   core developers (those with commit rights) should review all pull
   requests.  If you are the first to review a PR and approve of the
   changes use the GitHub `'approve review'
```

Comment:
I've moved both, the definition of core developers,, and the approve link into the initial paragraph, because they are not specific to *code changes*. I've also removed the distinction between tasks of first and second reviewer. Essentially, both approve, so there's no need to specify what each of the reviewers does. And the last one merges, which is now described in the final paragraph for all types of PRs.

Reply:
Is this comment at the right paragraph? There's nothing about self-merge in here anymore.